bokeh-demo/examples/landing_page/main.py

```diff
@@ -21,6 +35,7 @@ def __init__(self):
             self.data_manager.exams_df, self.data_manager.hpv_df
         )
         self._add_HR_screening_indicator()
+        self.n_total_people = format_number(len(self.person_df))
```

Comment:
Minor, but I think `n_total_people` makes it sound like it's an int ( and not a str ).